src/Symfony/Bridge/Twig/Extension/CodeExtension.php

```diff
@@ -98,7 +98,11 @@ public function formatArgs(array $args): string
             } elseif ('resource' === $item[0]) {
                 $formattedValue = '<em>resource</em>';
             } else {
-                $formattedValue = str_replace("\n", '', htmlspecialchars(var_export($item[1], true), \ENT_COMPAT | \ENT_SUBSTITUTE, $this->charset));
+                if (preg_match('//u', $item[1])) {
```

Comment:
when ppl don't use the UTF-8 charset, this might lead to false positives
what about checking using `preg_match('/[^\x00\x07-\x0d\x1B\x20-\xff]/', $value)` instead?

also, please unindent the if else by merging it with the surrouding one

Reply:
> also, please unindent the if else by merging it with the surrouding one

The parent `if` checks the value of `$item[0]` exclusively. The merged `else-if` would then check `$item[1]`. I explicitly wanted to reduce cognitive load here (having X-branches of an `if` checking the same variable only to have the last one do something different is not very readable in my opinion). If you insist I'll still merge the checks though.

Reply:
> If you insist I'll still merge the checks though.

We do. It's our code style.